source/translator.d

```diff
@@ -0,0 +1,35 @@
+private string[string] defaultLanguage;
+static this()
+{
+	defaultLanguage = [
+		"editor.run": "Run",
+		"editor.reset": "Reset",
+		"editor.format": "Format",
+		"editor.keyboard_shortcuts": "Keyboard Shortcuts",
+	];
+}
+
+/**
+Allows per-language overwrites of UI text
+*/
+class Translator
```

Comment:
Can you make this a `struct`? We can make it a `class` later if there's a need to do so.